Dockerfile

```
@@ -0,0 +1,7 @@
+FROM node:20
+
+COPY entrypoint.sh /entrypoint.sh
+COPY . /app
+RUN cd /app && npm install
```

Use WORKDIR to switch to a dir

```suggestion
WORKDIR /app
RUN npm install
```

---

Yes, this would be great for clarity. However, GitHub does something funny with `WORKDIR` when they run the container, and because of that [explicitly recommend](https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#workdir) that you do not set a `WORKDIR` in your Dockerfile.

> GitHub sets the working directory path in the GITHUB_WORKSPACE environment variable. It's recommended to not use the WORKDIR instruction in your Dockerfile.